src/Controls/src/Core/Platform/ModalNavigationManager/ModalNavigationManager.Android.cs

```diff
@@ -211,6 +211,9 @@ sealed class ModalContainer : ViewGroup
 				UpdateBackgroundColor();
 				AddView(_backgroundView);
 
+				// Don't let touch events pass through to the view being covered up
+				this.Touch += (_, args) => args.Handled = true;
```

Comment:
Are we good not unregistering this?

Reply:
The only reason to unregister is if subscribing to the platform events will cause memory leaks or other types of collection issues/delays.  These views are all single use for each modal that pops up, so this subscription isn't going to get hit multiple times and the platform modals container isn't reused.

Reply:
I switched this code over to just use `override TouchEvent`